packages/vue-components/src/Tabset.vue

```diff
@@ -2,7 +2,7 @@
   <div :class="[addClass]">
     <!-- Nav tabs -->
     <ul
-      class="nav nav-tabs"
+      class="nav nav-tabs no-print"
```

Comment:
We can actually use `d-print-none` here, so we don't need the print media query :-)